test-app/app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,8 @@
   </head>
 
   <body>
-    <%= yield %>
+    <div role="main">
```

Comment:
I want to say you should be able to use [`main`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) instead which might be more semantic.

Reply:
Fair point; I believe that is preferred over the use of the ARIA role.